docs/1 Introduction/1 Introduction.md

```diff
@@ -1,45 +1,56 @@
 # An introduction to Starport
 
-Let us dive into Starport, what we can achieve with it, and which other technologies play well with it.
+Starport is a developer-friendly interface to the Cosmos SDK.
 
 ## What is Starport?
 
-Starport is a tool that makes it easier to create blockchains.
+The Starport tool is the easiest way to create blockchains using the Tendermint Consensus engine and the Cosmos SDK. <!-- @fadeev does it matter that we use Go? --> Starport is written in the Go programming language. The new blockchain has a Proof-of-Stake (PoS) system with validators that can be defined in the genesis block.
 
-Starport uses the Tendermint Consensus engine and the Cosmos SDK to create a blockchain application in the Go programming language. This blockchain has a Proof-of-Stake system with validators that can be defined in the genesis block.
+With just a few commands, you can use Starport to:
 
-With just a few commands, you can create a blockchain, launch it, serve it on the cloud and have a GUI ready to start testing your application.
+1. Create a blockchain
+2. Launch a blockchain
+3. Serve your blockchain on the cloud
+4. Create a client-side app you can interact with in web browser
 
-Bootstrapping blockchains was initially the job of the `scaffold` program, which was used to create a blockchain application. Starport takes it to the next level and also creates a user interface with Vue.js, which provides a good starting point for developers creating a browser-based client-side application for your blockchain.
+<!-- do we need to mention Vue.js? -->
```

Comment:
We're actively working on the functionality to automatically create JS and Vuex clients for blockchains. Hopefully, it will be released very soon, so we can mention it in the docs. We're also workng on a UI in Vuex that provides users with commonly used components, and it's a rather important part of Starport.

Reply:
cool stuff, I have so much to learn. I have to look up every component for practically every paragraph. Thanks for your patience and thank you especially for teaching me @fadeev